media-libs/avidemux-core/files/avidemux-core-2.6.8-gcc6.patch

```diff
@@ -0,0 +1,50 @@
+--- a/avidemux_core/ADM_coreUtils/src/prefs.cpp
++++ b/avidemux_core/ADM_coreUtils/src/prefs.cpp
+@@ -44,8 +44,8 @@
+    const char *name2;
+    ADM_paramType  type;
+    const char *defaultValue;
+-   float    min;
+-   float    max;
++   double    min;
++   double    max;
+ }optionDesc;
+ 
+ #include "prefs2_pref.h"
+--- a/avidemux_core/ADM_coreAudio/src/ADM_audioStreamBuffered.cpp
++++ b/avidemux_core/ADM_coreAudio/src/ADM_audioStreamBuffered.cpp
+@@ -46,7 +46,7 @@
+         // By construction, the error should be minimal
+         if(newDts!=ADM_AUDIO_NO_DTS)
+         {
+-            if( abs(newDts-lastDts)>ADM_MAX_SKEW)
++            if( labs((int64_t)newDts-lastDts)>ADM_MAX_SKEW)
```

Comment:
Taken from part of upstream commit https://github.com/mean00/avidemux2/commit/657a31211973c6dfc483f1332a34c915d0026651